workers/unity/Assets/Gdk/Core/Utility/Bool.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+
+namespace Improbable.Gdk.Core
+{
+    public struct Bool
```

Comment:
This name is super issue prone!
It will be way too easy to use this elsewhere in code through auto complete or accident.

Reply:
Also make sure this implements the same interfaces and functions as the original bool1

Reply:
I vote for @jared-improbable 's suggestion of `BlittableBool`